art/estimators/certification/derandomized_smoothing/vision_transformers/vit.py

```diff
@@ -0,0 +1,196 @@
+# MIT License
```

Comment:
I think this file should following ART's naming convention be called pytorch.py

Reply:
renamed